integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java

```diff
@@ -460,7 +460,7 @@ private int renameInternal(String sourcePath, String destPath, int flags) {
     Optional<URIStatus> sourceStatus = AlluxioFuseUtils.getPathStatus(mFileSystem, sourceUri);
     if (!sourceStatus.isPresent()) {
       LOG.error("Failed to rename {} to {}: source non-existing", sourcePath, destPath);
-      return -ErrorCodes.EEXIST();
+      return -ErrorCodes.ENOENT();
```

Comment:
wondering how this error code gets checkin?

Reply:
Please check https://man7.org/linux/man-pages/man3/errno.3.html

- ENOENT No such file or directory
- EEXIST File exists